client/src/components/Visualization/charts/Geochart.js

```diff
@@ -1,59 +1,67 @@
 /**
  * Description: Geochart implements a Google geochart component.
  * Author: Kira Toal
- * Date: 2020/07/13
+ * Date: 2020/07/21
  */
-
-import React, { useEffect, useState } from 'react';
-import firebase, { app, database } from '../../../firebase/firebase';
-
-import { Chart } from 'react-google-charts';
-import { states } from './StateDataParser';
+import { Chart } from "react-google-charts";
+import React, { useState, useEffect } from 'react';
+import { STATES } from '../../../constants/geochart_constants';
+import { database } from '../../../firebase/firebase';
 
 const Geochart = () => {
+  const STATES_COLLECTION = 'dev_states';
   const [adTotal, setAdTotal] = useState(0);
```

Comment:
How does this work? 
I assume the second element is used to send data to the chart since you call it with your `dataTable` below.

What's the first element for?

Reply:
Ignore this. I understand now from looking at a different PR.